l10n_it_account_balance_eu/data/account_balance_eu_reclassification.xml

```diff
@@ -0,0 +1,967 @@
+            model="account.balance.eu"
+            name="account_balance_eu_debit_association"
+            eval="['1509%',ref('PA_C21a')]"
+        />
+        <function
+            model="account.balance.eu"
+            name="account_balance_eu_debit_association"
+            eval="['1510%',ref('PA_C21a')]"
+        />
+        <function
+            model="account.balance.eu"
+            name="account_balance_eu_debit_association"
+            eval="['1511%',ref('PA_C21a')]"
+        />
+        <function
+            model="account.balance.eu"
+            name="account_balance_eu_debit_association"
+            eval="['1531%',ref('PA_C25a')]"
+        />
+        <function
+            model="account.balance.eu"
+            name="account_balance_eu_debit_association"
+            eval="['1540%',ref('PA_C25a')]"
+        />
+        <function
+            model="account.balance.eu"
+            name="account_balance_eu_debit_association"
+            eval="['1541%',ref('PA_C25a')]"
+        />
+        <function
+            model="account.balance.eu"
+            name="account_balance_eu_debit_association"
+            eval="['1601%',ref('PA_C2Ba')]"
+        />
+        <function
+            model="account.balance.eu"
+            name="account_balance_eu_debit_association"
+            eval="['1602%',ref('PA_C2Ba')]"
+        />
+        <function
+            model="account.balance.eu"
+            name="account_balance_eu_debit_association"
+            eval="['1605%',ref('PA_C2Ba')]"
+        />
+        <function
+            model="account.balance.eu"
+            name="account_balance_eu_debit_association"
+            eval="['1607%',ref('PA_C2T')]"
+        />
+        <function
+            model="account.balance.eu"
+            name="account_balance_eu_debit_association"
+            eval="['1608%',ref('PA_C2Ba')]"
+        />
+        <function
+            model="account.balance.eu"
+            name="account_balance_eu_debit_association"
+            eval="['1609%',ref('PA_C2Ba')]"
+        />
+        <function
+            model="account.balance.eu"
+            name="account_balance_eu_debit_association"
+            eval="['1610%',ref('PA_C2Ba')]"
+        />
+        <function
+            model="account.balance.eu"
+            name="account_balance_eu_debit_association"
+            eval="['1620%',ref('PA_C25a')]"
+        />
+        <function
+            model="account.balance.eu"
+            name="account_balance_eu_debit_association"
+            eval="['1630%',ref('PA_C25a')]"
+        />
+        <function
+            model="account.balance.eu"
+            name="account_balance_eu_debit_association"
+            eval="['1640%',ref('PA_C25a')]"
+        />
+        <function
+            model="account.balance.eu"
+            name="account_balance_eu_debit_association"
+            eval="['1701%',ref('PA_C36')]"
+        /> <!-- conto non presente sul PdC base, MKT ha i "titoli" in 130102-->
```

Comment:
Andrebbe in inglese. Sembra comunque un commento "interno" di MKT, in tal caso andrebbe tolto.

Reply:
:+1: